<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipHangar</class>
 <widget class="QMainWindow" name="ShipHangar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ship Hangar</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>rsc/starclitz-logo.ico</normaloff>rsc/starclitz-logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutHeader">
      <item>
       <widget class="QOpenGLWidget" name="openGLWidgetShipView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutShipSelection">
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxVariant"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select Variant:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select model:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelMake">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Select make:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxMake"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxModel"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>V Add to Hangar V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidgetShips">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
